### PR TITLE
Update CPS version

### PR DESCRIPTION
**Pull request by dmonroym**

### CODE REVIEW

File: build/import/Packages.targets [hunk 1]
```diff
@@ -105,13 +105,13 @@
...
-    <PackageReference Update="Microsoft.VisualStudio.ProjectSystem.SDK"                               Version="16.2.133-pre" />
-    <PackageReference Update="Microsoft.VisualStudio.ProjectSystem.Analyzers"                         Version="16.2.133-pre" />
+    <PackageReference Update="Microsoft.VisualStudio.ProjectSystem.SDK"                               Version="16.4.28-pre" />
+    <PackageReference Update="Microsoft.VisualStudio.ProjectSystem.Analyzers"                         Version="16.4.28-pre" />
 
     <!-- Roslyn -->
     <PackageReference Update="Microsoft.VisualStudio.LanguageServices"                                Version="3.2.0-beta4-19359-03" />
     <PackageReference Update="Microsoft.CodeAnalysis"                                                 Version="3.2.0-beta4-19359-03" />
-    <PackageReference Update="Microsoft.VisualStudio.IntegrationTest.Utilities"                       Version="2.6.0-beta1-62113-02" />
+    <PackageReference Update="Microsoft.VisualStudio.IntegrationTest.Utilities"                       Version="2.6.0-beta1-62113-02" />Microsoft.VisualStudio.Threading
```
Comment (davidwengier) [inline build/import/Packages.targets]: Typo?

Comment (dmonroym) [reply build/import/Packages.targets]: Yes, thanks

**Comment by dmonroym** [comment]
Updating versions now generates 166 Errors most of which are related to nullability such as:
CS8600: Converting null literal or possible null value to non-nullable type.
CS8602: Dereference of a possibly null reference.
CS8603: Possible null reference return.

**Comment by abpiskunov** [comment]
@vijayrkn @BillHiebert  FYI

**Comment by dmonroym** [comment]
This actually is already being taken care of in #5506 so I'll abandon this one.

@abpiskunov you should track Drew's PR instead.

**Pull request closed without merging**